lib/hammer_cli_foreman_leapp.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+require 'hammer_cli'
+require 'hammer_cli_foreman'
+
+require 'hammer_cli_foreman_leapp/version'
+require 'hammer_cli_foreman_leapp/preupgrade_reports'
+
+module HammerCLIForemanLeapp
+  HammerCLI::MainCommand.lazy_subcommand(
+    'preupgrade_reports',
+    'Manage Leapp preupgrade_reports',
```

Comment:
I guess this is what gets displayed to the user, right? If that is the case, then we should probably drop the underscore from preupgrade_report